app/presenters/transaction_presenter.rb

```diff
@@ -1,4 +1,7 @@
 class TransactionPresenter < ContentItemPresenter
+  attr_writer :variant_slug
+  attr_reader :variant_slug
```

Comment:
You can use `attr_accessor : variant_slug` here rather than the separate `attr_writer` and `attr_reader`.